assets/src/scripts/lib/compound.js

```diff
@@ -0,0 +1,47 @@
+export default function compound() {
+  var scales = [].slice.call(arguments);
```

Comment:
I'm just curious about the `var` vs `let` here?

Also, it seems that this stylistically a bit different from the rest of the javascript... :thinking:

Reply:
This is a copy-paste of the code I submitted to d3-scale, so I wrote it in the D3 style.

Reply:
Ah. Makes sense.